connection.go

```diff
@@ -46,7 +46,7 @@ func (e *ErrUnpack) Unwrap() error {
 // Connection represents an ISO 8583 Connection. Connection may be used
 // by multiple goroutines simultaneously.
 type Connection struct {
-	addr           string
+	Addr           string
```

Comment:
Why are we exporting this?

Reply:
This section was a carbon copy of the area from @alovak's PR - can probably un-export, figured I would try to ease his merge conflict =D

Reply:
We are using Addr in logs to distinguish connections. Not having access to Addr makes it’s hard

Reply:
Ah okay. That's fine, but it wasn't part of "add connection timeout."